src/sentry/tasks/embeddings_grouping/utils.py

```diff
@@ -487,3 +512,12 @@ def delete_seer_grouping_records(
         for group in groups_with_seer_metadata:
             del group.data["metadata"]["seer_similarity"]
         Group.objects.bulk_update(groups_with_seer_metadata, ["data"])
+
+
+def get_project_for_batch(last_processed_project_index, cohort_list, cohort_name):
+    next_cohort_index = last_processed_project_index + 1
+    if next_cohort_index >= len(cohort_list):
+        return None, None
+    project_id = cohort_list[next_cohort_index]
+    last_processed_project_index = next_cohort_index
+    return project_id, last_processed_project_index
```

Comment:
`cohort_name` doesn't seem like it's used.

Naming nits:

```suggestion
def get_project_for_batch(last_processed_project_index, cohort_projects, cohort_name):
    next_project_index = last_processed_project_index + 1
    if next_project_index >= len(cohort_projects):
        return None, None
    project_id = cohort_projects[next_project_index]
    last_processed_project_index = next_project_index
    return project_id, last_processed_project_index
```

Reply:
yup. thanks